integration_tests/__tests__/__snapshots__/transform-test.js.snap

```diff
@@ -0,0 +1,27 @@
+exports[`babel-jest instruments only specific files and collects coverage 1`] = `
+"Using Jest CLI v14.0.0, jasmine2, babel-jest
```

Comment:
one thing that kinda sucks is that for every version bump, this test will fail. Since we don't run the tests between npm publish and a git push, we'll have to manually do it. How about changing this test to exclude the first line, since it isn't significant?